apps/admin-portal/src/components/roles/edit-role/edit-role-basic.tsx

```diff
@@ -85,11 +85,20 @@ export const BaiscRoleDetails: FunctionComponent<BasicRoleProps> = (props: Basic
      * @param values Form values which will be used to update the role
      */
     const updateRoleName = (values: any): void => {
-        const roleData: CreateRoleInterface = {
-            displayName: values.get("rolename")
+        const roleData: PatchRoleData = {
+            schemas: [
+                "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+            ],
+            Operations: [{
+                "op": "replace",
+                "value": {
+                    "displayName": values.get("rolename")
+                }
+            }]
         }
 
-        updateRoleDetails(roleId, roleData).then(response => {
+        updateRoleDetails(roleObject.id, roleData).then(response => {
```

Comment:
Shall we bring `then` to a new line and handle `catch` as well.

Reply:
fixed with commit 0482c99